### PR TITLE
Milestone #4: Add Styling

### PR DESCRIPTION
**Pull request by Anjacodes**
In this milestone, I:

* Used the information detailed in Zeplin to style the website.
* Matched the final design to Zeplin's design.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+          <Link className={styles.link} to="/">BOOKS</Link>
+        </li>
+        <li className={styles.linkEl}>
+          <Link className={styles.link} to="/categories">CATEGORIES</Link>
+        </li>
+      </ul>
+    </div>
+    <img className={styles.user} alt="user account logo" src={user} />
   </nav>
 );
```
Comment (Emmyn5600) [inline src/components/Navbar.js:20]: - [ ] Hi, good job @Anjacodes however let us try to meet the requirements as well as the design provided. As you can see the menu or navbar div at the original design has the same width as the div or card that has the information added. 

Original design

header

Your design

header2

File: src/components/AddBook.js [hunk 2]
```diff
@@ -23,24 +24,26 @@ const AddBook = () => {
...
           value={category}
           onChange={(e) => setCategory(e.target.value)}
+          className={styles.category}
           required
         />
-        <button type="submit" onClick={(e) => submitBookToStore(e)}>Add Book</button>
+        <button className={styles.btn} type="submit" onClick={(e) => submitBookToStore(e)}>ADD BOOK</button>
       </form>
     </div>
   );
```
Comment (Emmyn5600) [inline src/components/AddBook.js]: - [ ] You can also check the design of the add book button comparing yours and the original design.

Original design

design

Your design

design2

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @Anjacodes 

Great work on making the project requirements 👏 
You've done well implementing some of the project requirements, but there are still some which aren't addressed yet, please work on it and ask for another review.

 ## Required Changes ♻️

- Check the comment under the review.

## Optional suggestions

- n/a

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @Anjacodes, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

tenor (1)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.js (hunk 2 above)
Comment (Anjacodes) [reply src/components/AddBook.js]: Thank you @Emmyn5600, I have made sure that the Add Book section is as wide as the rest.

**Comment by Anjacodes** [comment]
Thank you Brian!

**Pull request merged**